src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -413,10 +429,78 @@ private void Initialize()
 
             Focusable = true;
             KeyEvent += OnKeyEvent;
+
+            Accessibility.Accessibility.Enabled += (s, e) => {
+                isAtspiEnabled = true;
+                AccessibilityEnabled();
+            };
+            Accessibility.Accessibility.Disabled += (s, e) => {
+                isAtspiEnabled = false;
+                HideItemsForAccessibility();
+            };
+            Accessibility.Accessibility.ScreenReaderEnabled += (s, e) => {
+                isScreenReaderEnabled = true;
+                AccessibilityEnabled();
```

Comment:
This ```AccessibilityEnabled()``` and ```HideItemsForAccessibility()``` of line 447 are not neccessary,
because there is not the case that the ```IsScreenReaderEnabled``` is ```TRUE``` and ```IsEnabled``` is ```FALSE```.
i.e. If the ```IsScreenReaderEnabled``` is ```TRUE``` then the ```IsEnabled``` should be ```TRUE```.

Reply:
Thank you for the clear explanation.